Centralize soc_to_chipset map

cccclai opened the pull request:
Summary: We have multiple copy of the maps in the codebase and it's error prone. Provide a util function for the map so we can get the map from the same function

Differential Revision: D64080089

haowhsu-quic reviewed (approved):
That's nice, thank you for helping this.

cccclai committed:
Centralize soc_to_chipset map (#6033)

Summary:

We have multiple copy of the maps in the codebase and it's error prone. Provide a util function for the map so we can get the map from the same function

Reviewed By: kirklandsign

Differential Revision: D64080089
